src/main/java/bizbook/model/AddressBook.java

```diff
@@ -92,6 +98,52 @@ public void setPerson(Person target, Person editedPerson) {
      */
     public void removePerson(Person key) {
         persons.remove(key);
+        if (isPinned(key)) {
+            removePinnedPerson(key);
+        }
+    }
+
+    //=========== Pinned Person List Accessors ===============================================================
+
+    /**
+     * Returns true if a person with the same identity as {@code person} exists in the pinned list.
+     */
+    public boolean isPinned(Person person) {
+        requireNonNull(person);
+        return pinnedPersons.contains(person);
+    }
+
+    /**
+     * Returns an unmodifiable view of the pinned person list.
+     */
+    public ObservableList<Person> getPinnedPersonList() {
+        return pinnedPersons.asUnmodifiableObservableList();
+    }
+
+    /**
+     * Replaces the contents of the pinned list with {@code persons}.
+     * {@code persons} must not contain duplicate persons.
+     */
+    public void setPinnedPersons(List<Person> persons) {
+        this.pinnedPersons.setPersons(persons);
+    }
+
+    /**
+     * Adds the {@code person} in the pinned contact list.
+     *
+     * @param person The {@code person} in the contact list to be pinned.
+     */
+    public void addPinnedPerson(Person person) {
+        this.pinnedPersons.add(person);
+    }
+
```

Comment:
Could you add the "pinnedPersons.equals(otherAddressBook.pinnedPersons);" to the equals method, it would be needed the revised implementation of "commit" i stated above.

Reply:
Same as what was mentioned above

Reply:
Still need the pinnedPersons equals to be implemented.